raiden-network/light-client
# Introduces pnpm

**Thank you for submitting this pull request :)**

Fixes # 

**Short description**


**Definition of Done**


**Steps to manually test the change (dApp)**

- Remove the previous `node_modules` directories for `raiden-ts`, `raiden-dapp` and `raiden-cli`
- run `npm i -g pnpm` to install pnpm
- In the root of the project run `pnpm install` this should install the whole workspace
- to serve the dApp run `pnpm run serve --filter raiden-dapp`
- to run lint for the whole project you can run `pnpm run lint -r`
- to run tests for the sdk you can do a `pnpm run test --filter raiden-ts`
